### PR TITLE
feat(cli): adjust `prisma init` to write to .env instead of prisma/.env

### LINKED ISSUE #3721: Adjust `prisma init` to write to .env instead of prisma/.env
## Problem

By default when we initialize a Prisma project, we write an environment file to `prisma/.env`. This layout works for us but is unusual and incompatible with projects (e.g. next.js, blitz, redwood, create-react-app projects) that assume `.env` is in the project root.

If you try moving `prisma/.env` into the project root, you'd also need to move `prisma/schema.prisma` to the project root. This approach is [documented](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema#using-env-files), but subtle.

For context, there are two components that rely on `.env`: 

- The CLI when introspecting and migrating.
- The generated database client when connecting to the database.

## Suggested solution

I'd suggest we change our defaults from `prisma/.env` and `prisma/schema.prisma` to `.env` and `schema.prisma`. If someone already has an `.env` file, we append our default environment variable (`DATABASE_URL`) to the bottom of their existing `.env` file.

### Do nothing

Don't change anything. Add a bit of documentation saying how you can make Prisma work with your existing `.env` file.

### Initialize to `.env` and `prisma/schema.prisma`

`prisma init` could also write `.env` and `prisma/schema.prisma` instead of writing both files to the project root. I don't really see a reason to create a subdirectory with only 1 file (by default), but I also don't have strong opinions on this.

## Related

- We should decide on #3720 before deciding on this issue.

### PR DESCRIPTION
**Pull request by williamluke4**
- [x] Add tests

### CODE REVIEW

File: src/packages/cli/src/Init.ts [hunk 1]
```diff
@@ -73,7 +74,6 @@ export class Init implements Command {
     }
 
     const outputDir = process.cwd()
-    const prismaFolder = path.join(outputDir, 'prisma')
```
Comment (timsuchanek) [inline src/packages/cli/src/Init.ts:76]: Why don't you check for the prisma dir anymore?

File: src/packages/cli/src/Init.ts [hunk 2]
```diff
@@ -138,15 +127,23 @@ export class Init implements Command {
...
     )
-    fs.writeFileSync(path.join(prismaFolder, '.env'), defaultEnv(url))
+    const envPath = path.join(outputDir, '.env')
+    if (!fs.existsSync(envPath)) {
+      fs.writeFileSync(envPath, defaultEnv(url))
+    }  else {
+      const envFile = fs.readFileSync(envPath, { encoding: 'utf8'})
+      const config = dotenv.parse(envFile) // will return an object
+      if(Object.keys(config).includes("DATABASE_URL")){
+        console.warn(`${chalk.yellow('warn')} DATABASE_URL already exists in ${chalk.bold(envPath)}`)
```
Comment (timsuchanek) [inline src/packages/cli/src/Init.ts:141]: Let's be more actionable here and tell users what we would have printed and what they can do

File: src/packages/cli/src/Init.ts (hunk 1 above)
Comment (williamluke4) [reply src/packages/cli/src/Init.ts:76]: Didn't think it was necessary as we are moving away from that convention. I'll re-add the check if you think it is necessary

File: src/packages/cli/src/Init.ts (hunk 2 above)
Comment (williamluke4) [reply src/packages/cli/src/Init.ts:141]: What do you mean by this? This is the information that is currently printed https://github.com/prisma/prisma/blob/williamluke4/issue3721/src/packages/cli/src/__tests__/__snapshots__/init.test.ts.snap#L49-L66

Comment (timsuchanek) [reply src/packages/cli/src/Init.ts:141]: Something like:
```
To resolve this, we recommend moving the `DATABASE_URL` env var to the root .env
```

File: src/packages/cli/src/Init.ts (hunk 1 above)
Comment (timsuchanek) [reply src/packages/cli/src/Init.ts:76]: That is still our convention, but users can customize.
We'll generate into `prisma/schema.prisma`, so we need to check if the file is already there.

File: src/packages/cli/src/Init.ts (hunk 2 above)
Comment (williamluke4) [reply src/packages/cli/src/Init.ts:157]: They don't really need to resolve anything as this is just a warning them that they already have an ENV VAR named DATABASE_URL in their root `.env`. Therefore we will just not add our default DATABASE_URL to it.

**Pull request closed without merging**